sim/core/spell_result.go

```diff
@@ -603,10 +603,14 @@ func (spell *Spell) WaitTravelTime(sim *Simulation, callback func(*Simulation))
 
 // Returns the combined attacker modifiers.
 func (spell *Spell) AttackerDamageMultiplier(attackTable *AttackTable, isPeriodic bool) float64 {
+	cumulativeAdditiveMultiplier := float64(TernaryInt64(isPeriodic,
+		spell.DamageMultiplierAdditivePct+spell.PeriodicDamageMultiplierAdditivePct,
+		spell.DamageMultiplierAdditivePct+spell.ImpactDamageMultiplierAdditivePct,
+	)) / 200
+
 	return spell.attackerDamageMultiplierInternal(attackTable) *
 		spell.DamageMultiplier *
-		spell.DamageMultiplierAdditive *
-		TernaryFloat64(isPeriodic, spell.PeriodicDamageMultiplierAdditive, spell.ImpactDamageMultiplierAdditive)
+		cumulativeAdditiveMultiplier
```

Comment:
The damage formula for this was WRONG. Additive multipliers need to be added with one another, not multiplying the base additive by the impact/periodic like we were. This causes all DPS specs to lose damage.